usage/test-framework/src/main/java/org/apache/brooklyn/test/framework/SimpleCommandDriver.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.brooklyn.test.framework;
+
+import org.apache.brooklyn.api.entity.EntityLocal;
+import org.apache.brooklyn.api.entity.drivers.EntityDriver;
+import org.apache.brooklyn.api.location.Location;
+
+import java.util.Collection;
+
+/**
+ * Driver to invoke a command on a node.
+ */
+public interface SimpleCommandDriver extends EntityDriver {
+
+    /**
+     * Result of the command invocation.
+     */
+    interface Result {
```

Comment:
Could think about the relationship between this and `org.apache.brooklyn.feed.ssh.SshPollValue`. Feels like at some point we should extract a common interface/class, but not sure exactly where yet (so don't worry about it for this PR).